module/VuFind/src/VuFind/Db/Service/UserService.php

```diff
@@ -43,11 +45,20 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  */
-class UserService extends AbstractDbService implements LoggerAwareInterface, DbServiceAwareInterface
+class UserService extends AbstractDbService implements
+    LoggerAwareInterface,
```

Comment:
You seem to have dropped DbServiceAwareInterface here. It's probably not needed yet, but I included it because it will be needed, and I'm trying to keep things parallel with #2233. However, if you prefer to remove it for now, that's okay, but in that case, you should also remove DbServiceAwareTrait below for consistency.

Reply:
I think I dropped it per your earlier comment but happy to put it back!

Reply:
Sorry, you're right about that. In any case, then you should drop the associated trait as well. We can re-add in #2233 if we actually need it.